data/scripts.js

```diff
@@ -1028,6 +1701,21 @@ let BattleScripts = {
 	targetTypeChoices(targetType) {
 		return CHOOSABLE_TARGETS.has(targetType);
 	},
+
+	reduceMoveResults(states) {
```

Comment:
`reduce` is a bad name which I suspect you just chose because you run the reduce function on the results.

```suggestion
	combineResults(states) {
```

is what I'd call it.